LocalWiki.php

```diff
@@ -196,6 +196,67 @@ function onBeforePageDisplay( OutputPage $outputPage ) {
 		$wgSpecialPages['Analytics'] = DisabledSpecialPage::getCallback( 'Analytics', 'MatomoAnalytics-disabled' );
 		$wgPageImagesScores['position'] = [ 100, -100, -100, -100 ];
 
+		break;
+	case 'kagagawiki':
+		$wgUploadWizardConfig = [
+			'uwLanguages' => [
```

Comment:
Should also probably include the default value we have set here also:
```suggestion
			'campaignExpensiveStatsEnabled' => false,
			'flickrApiKey' => $wmgUploadWizardFlickrApiKey,
			'uwLanguages' => [
```

Reply:
Thank you for your reply.

> `'campaignExpensiveStatsEnabled' => false,`

I don't know what is the value for.
Is this for decrease computer resources?

https://github.com/wikimedia/mediawiki-extensions-UploadWizard/blob/master/UploadWizard.config.php#L109C2-L110C42

> `'flickrApiKey' => $wmgUploadWizardFlickrApiKey,`

We'er not using Flickr, still we should set the API key?

Reply:
> Thank you for your reply.
> 
> > `'campaignExpensiveStatsEnabled' => false,`
> 
> I don't know what is the value for. Is this for decrease computer resources?

Yep, this should always be disabled no matter what
> https://github.com/wikimedia/mediawiki-extensions-UploadWizard/blob/master/UploadWizard.config.php#L109C2-L110C42
> 
> > `'flickrApiKey' => $wmgUploadWizardFlickrApiKey,`
> 
> We'er not using Flickr, still we should set the API key?

It may cause warnings on our side if this isn't set while unlikely it will cause issues it is possible so please do set it anyway.

Reply:
Okay, I added both.